frontend/src/components/ActivityCard/ActivityCard.js

```diff
@@ -1,22 +1,45 @@
 import React, { Component } from 'react';
 import { Card, CardContent , CardMedia, Typography, CircularProgress } from '@material-ui/core';
+import TagChip from '../TagChip/TagChip';
 import './ActivityCard.css';
 
 class ActivityCard extends Component {
   render() {
-    const { logo, title, tags, name, error, isLoading } = this.props;
-    let contentInfo;
+    let { logo, title, tags, name, error, isLoading } = this.props;
+    let contentInfo, sortedTags = [];
 
     if (isLoading) {
       contentInfo = 'Loading data';
     } else if (!error) {
-      contentInfo = <ul>
-        <li>Name: { name || '(not set)'}</li>
-        <li>Languages: { tags && tags.length ?
-          tags.map((item, i) => <span key={i}>{item} </span>) :
-          'no languages'
-        }</li>
-      </ul>;
+      if (tags  && Object.keys(tags).length) {
+        sortedTags = Object.keys(tags);
+        sortedTags.sort((a, b) => {
+          if (tags[a] < tags[b]) {
+            return 1;
+          }
+          else {
+            return -1;
+          }
+        });
+      }
+
+      contentInfo = (<div>
+        <p>Name: { name || '(not set)'}</p> 
+        <div>
+          { tags && Object.keys(tags).length ?
+              sortedTags.map((key, i) => 
+                <TagChip
+                  key={i}
+                  tag={key}
+                  real={Math.floor(tags[key]*100)}
+                  value={Math.floor(tags[key]*100) ? Math.floor(tags[key]*100) : '>1'}
```

Comment:
Should be `<1` if I understand it correctly.

Reply:
++